Ago-Dic-2020/perez-sanchez-jose-jahir/Practica2/Practica2.py

```diff
@@ -0,0 +1,38 @@
+class Animal():
+    def __init__(self, edad, tamaño, peso, altura):
+        self.edad = edad
+        self.tamaño = tamaño
+        self.peso = peso
+        self.altura = altura
+
+    def __str__(self):
+        return f"Edad: {self.edad} años\nTamaño: {self.tamaño}\nPeso: {self.peso}kg\nAltura: {self.altura}cm"
+    
+    def IMC(self):
```

Comment:
Las funciones y los nombres de variables deben de estar en minúsculas: https://www.python.org/dev/peps/pep-0008/#function-and-variable-names